jwst/extract_1d/soss_extract/soss_extract.py

```diff
@@ -508,24 +518,18 @@ def model_image(scidata_bkg, scierr, scimask, refmask, ref_files, box_weights, s
     refmask : array[bool]
         Pixels that should never be reconstructed e.g. the reference pixels.
     ref_files : dict
-        A dictionary of the reference file DataModels.
+        A dictionary of the reference file DataModels, along with values for
+        subarray and pwcpos, i.e. the pupil wheel position.
     box_weights : dict
         A dictionary of the weights (for each order) used in the box extraction.
         The weights for each order are 2d arrays with the same size as the detector.
-    subarray : str
-        Subarray on which the data were recorded; one of 'SUBSTRIPT96',
-        'SUBSTRIP256' or 'FULL'.
-    transform : array or list, optional
```

Comment:
Not sure the best place to put this comment, but it looks like `soss_kwargs["transform"]` is no longer used anywhere in the code.  Can `soss_transform` safely be removed from the arguments to `extract_1d_step`?  If so, don't forget to update the docs to reflect that.

Reply:
I've attempted to fully excise this parameter from both code and the documentation.